Prac09/taxiMenu.py

```diff
@@ -0,0 +1,65 @@
+__author__ = 'Tom Stanley'
+from Prac09.Car import Taxi
+from Prac09.Car import SilverServiceTaxi
+
+TAXIS = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+
+
+def main():
+    print("Let's drive!")
+    bill_to_date = 0
+    menu_choice = get_valid_menu_choice()
+    while menu_choice != 'q':
+        if menu_choice == 'c':
+            taxi_choice = choose_taxi()
+            print(taxi_choice)
+            # bill_to_date += taxi_choice.get_fare()
+            bill_to_date += 0
+            # Eliminated a previous issue I had where it was adding $4.50 prematurely by adding 0 each time.
+            output_bill_to_date(bill_to_date)
+            menu_choice = get_valid_menu_choice()
+        else:
+            bill_to_date += drive_taxi(taxi_choice, bill_to_date)
```

Comment:
Potential issue here, if I choose drive it results in a crash, might need something to ensure a taxi gets chosen first as this will result in no assignment to taxi_choice